contracts/.env.template

```
@@ -1,2 +1,2 @@
-INFURA_ID=zzzzzzzzzzzzzzzzzzzzzzzzzzzzzzzz
+INFURA_ID=yourInfuraApiKey
```

there was some confusion earlier about "infura api key" vs "infura project id". I might recommend:

```suggestion
INFURA_ID=yourInfuraProjectId
```